src/CMakeLists.txt

```diff
@@ -91,10 +91,6 @@ generate_export_header(exiv2lib
     STATIC_DEFINE exiv2lib_STATIC
 )
 
-target_compile_features(exiv2lib_int PRIVATE cxx_std_98)
```

Comment:
There are some other usages of `target_compile_features` in the unitTests and samples folders.

Reply:
I've got rid of all the `target_compile_features` magic.